### PR TITLE
fix no bounty label

### PR DESCRIPTION
**Pull request by qiweiii**
## Description

No bounty added label

## Changes

If value is 0 in tasks list, it displays 'No bounty added'

## Issue

> #536 

## Impacted Area

> Main page

## Steps to test

View on tasks page

### CODE REVIEW

File: frontend/src/components/task/task-table.js [hunk 1]
```diff
@@ -256,8 +256,8 @@ class CustomPaginationActionsTable extends React.Component {
                         </div>
                       </TableCell>
                       <TableCell numeric style={ { padding: 5 } }>
-                        <div style={ { width: 40 } }>
-                          { n.value ? `$ ${n.value}` : '$ 0' }
+                        <div style={ { width: 70, textAlign: 'center' } }>
+                          { n.value ? (n.value === '0' ? 'No bounty added' : `$ ${n.value}`) : 'No bounty added' }
```
Comment (alexanmtz) [inline frontend/src/components/task/task-table.js]: You need to add the translation strings here. Take a look at our ReactIntl implementation.

**Comment by alexanmtz** [comment]
@qiweiii please provide a screenshot of your solution, how it looks like in your change request.

File: frontend/src/components/task/task-table.js (hunk 1 above)
Comment (qiweiii) [reply frontend/src/components/task/task-table.js]: @alexanmtz I made a new commit.

the screenshot after change:
image

**Comment by alexanmtz** [comment]
Great @qiweiii , now we have the second step for this issue, which is add the _no bounty_ on task page and add the buttons as the screenshot

**Comment by qiweiii** [comment]
@alexanmtz I added the no bounty label and add bounty button to task page. But I did not add 'make an offer' button as shown in the screenshot, I am not sure whether to add it. It seems 'make an offer' should only appear when the current viewer is not the owner of the task.

The screenshots after change:
image

image

**Comment by alexanmtz** [comment]
@qiweiii yeah, you're right, it's only available for other users, but the currently make an offer button is right down that card, it's just moving from the place we have today as here:
Screen Shot 2020-03-28 at 11 58 23 AM

To the side of the Add bounty.

I hope it's clear and it seems we're almost there 🤞

**Comment by qiweiii** [comment]
@alexanmtz please check the new commit. 

It now looks like this:
image

**Comment by alexanmtz** [comment]
@qiweiii great, thanks for send your changes requested. Have you check the tests? It is failing in the build, and other last thing to complete the translation flow is by add that translation entry in this file:
https://github.com/worknenjoy/gitpay/blob/master/locales/en.json

**Comment by qiweiii** [comment]
@alexanmtz I added translation to `en.json`.

**Comment by alexanmtz** [comment]
Great @qiweiii , I think we can merge now 🚀

**Pull request merged**